### ISSUE TITLE
stop booster command

### ISSUE THREAD

**Issue by Dolqhix** [Auxilor/Boosters#2]
a command to disable a booster.

**Comment by good3n** [comment]
> a command to disable a booster.

Wouldn't `/boosters cancel` do that?

**Comment by Dolqhix** [comment]
yes, i wrote this when the plugin was brand new and it didn't exist yet :)

**Comment by good3n** [comment]
> yes, i wrote this when the plugin was brand new and it didn't exist yet :)

Ok gotchya. If you could close the issue that would help keep feature requests cleaned up :)
